evergreen.yml

```diff
@@ -506,6 +557,60 @@ functions:
         bucket: translators-connectors-releases
         permissions: public-read
         content_type: application/octet-stream
+    - command: s3.get
```

Comment:
[nit] I believe that you can move multiple files at once for the s3 commands.
Listing all files one by one still work for me though. No need to change anything.

Reply:
Yes, there is a way to do that. I think I'll add this to the evergreen cleanup ticket.